docs/source/author/topics.rst

```diff
@@ -288,9 +288,21 @@ exercise, e.g. ``count-to-16``.
     4. Example solution(s)
 
       - **Is called:** ``<language>-solution.md`` where ``<language>`` is the key
-        of the programming language, e.g. ``ruby-hints.md``.
+        of the programming language, e.g. ``ruby-solution.md``.
       - **Contains:** Example solutions to the exercise, e.g. Scratch program.
 
+    5. Extra challenge(s) (optional)
+
+      - **Is called:** the value defined in the programming exercises
+        configuration file.
+        A common filename is ``extra-challenge.md``.
+      - **Contains:** Content for an extra challenge.
+
+      .. note::
```

Comment:
if you put this note here, you have to put it on every other md file that doesn't have a heading...